<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowLayout</class>
 <widget class="QMainWindow" name="MainWindowLayout">
  <property name="windowTitle">
   <string>QtOpenGL - Eberty Alves</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="file_label_">
      <property name="text">
       <string>File:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="file_line_edit_">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Please, select an OBJ file.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="open_button_">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string> Open</string>
      </property>
      <property name="icon">
       <iconset theme="folder">
        <normaloff/>
       </iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QtOpenGL" name="opengl_widget_" native="true">
      <property name="minimumSize">
       <size>
        <width>275</width>
        <height>175</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtOpenGL</class>
   <extends>QWidget</extends>
   <header>qt_opengl.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
